.github/workflows/container-tests.yml

```diff
@@ -17,22 +17,42 @@ jobs:
             os_test: "fedora"
             context: "Fedora"
             compose: "CentOS-7"
+            api_key: "TF_PUBLIC_API_KEY"
+            branch: "main"
+            tmt_repo: "https://github.com/sclorg/sclorg-testing-farm"
+            tf_scope: "public"
```

Comment:
This is not needed, I guess. The default value is public.

Reply:
Yes, but we take `tf_scope` input from the matrix. And in the matrix, there is no default. So IMHO empty value would be passed to the action, or it will fail.

Reply:
Undefined `tf_scope` in the matrix works properly. See https://github.com/sclorg/s2i-base-container/pull/209.
And the reason is simple. Only if `tf_scope` is defined as `private` then `osci...` is used otherwise `public` URL.